src/engraving/rw/xmlreader.cpp

```diff
@@ -220,9 +220,6 @@ String XmlReader::readXml()
 {
     static int count = 0;
     ++count;
-    if (count == 6) {
-        int k = -1;
-    }
```

Comment:
fixed here https://github.com/musescore/MuseScore/pull/11967
(`static int count = 0;` need to remove also)